### DISCUSSION TITLE
Unhandled Trolling in Issues

### DISCUSSION THREAD

**Discussion by BraINstinct0** [Open-Shell/Open-Shell-Menu#821] [General]
(will not mention name, but) There's a user who repeatedly post comments which are rather offensive/unrelated/gibberish-y.
Some (especially considering that this app is used more by non tech-savvy WIndows users) who posted issues to ask for help seem to be distracted/misled by the comment. Is there any way to ban this behavior?

**Comment by coddec** [comment]
For the time being maybe give this a try?
See the screenshot for reference only
![image](https://user-images.githubusercontent.com/12678338/131341650-5883e003-353b-4a2f-b3ab-c3a6efd306fd.png)
